### PR TITLE
Feature/water quality sensor

### PR DESCRIPTION
**Pull request by ArtemOliinyk**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -14,22 +14,30 @@
   ],
   "dependencies": {
     "body-parser": "~1.19.0",
+    "bootstrap": "~4.3.1",
+    "chart.js": "^2.8.0",
```
Comment (undead404) [inline client/package.json]: Use tilde notation please

File: client/src/components/Menu/index.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        <div className={s.menuHamburger}>
+          <Hamburger
+            isOpen={isMenuOpen}
+            menuClicked={this.onMenuClick}
+            width={25}
+            height={15}
+            color="#3f496f"
+          />
+        </div>
+        <div className={`${s.menuItems} ${isMenuOpen && s.menuItemsActive}`}>
```
Comment (undead404) [inline client/src/components/Menu/index.js]: use `classnames` library here please

File: client/src/components/Menu/index.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+          ))}
+        </div>
+        <div className={s.menuSocial}>
+          <a
+            href="https://google.com"
+            target="_blank"
+            rel="noreferrer noopener"
+            className={s.menuSocialIcon}
+          >
+            <svg
```
Comment (undead404) [inline client/src/components/Menu/index.js]: please extract your static SVGs to separate SVG files

File: client/src/components/Menu/index.js [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+            >
+              <path d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z" />
+            </svg>
+          </a>
+        </div>
+      </div>
+    );
+  }
+
+  menuItems = [
```
Comment (undead404) [inline client/src/components/Menu/index.js]: maybe extract these items into a separate JSON file or separate JS file.

**Review by undead404 — CHANGES_REQUESTED** [review]

File: client/package.json [hunk 5]
```diff
@@ -14,22 +14,30 @@
...
     "body-parser": "~1.19.0",
+    "bootstrap": "~4.3.1",
+    "chart.js": "~2.8.0",
+    "classnames": "~2.2.6",
     "cookie-parser": "~1.4.4",
     "core-js": "~3.2.1",
     "express": "~4.17.1",
     "history": "~4.7.0",
     "isomorphic-style-loader": "~5.1.0",
+    "moment": "^2.24.0",
```
Comment (undead404) [inline client/package.json]: Please use tilde in stable dependencies

File: client/src/components/LineChart/LineChart.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
+import React, { PureComponent } from 'react';
+import Chart from 'chart.js';
+import withStyles from 'isomorphic-style-loader/withStyles';
+import PropTypes from 'prop-types';
+import style from '../WaterQualitySensor/WaterQualitySensor.css';
```
Comment (undead404) [inline client/src/components/LineChart/LineChart.js]: Do you really need to share these styles among components?

File: client/src/components/WaterQualitySensor/WaterQualitySensor.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+        </Row>
+      </Container>
+    );
+  }
+
+  filterData = query => {
+    console.info(1, query);
+  };
+
+  loading = () => {
```
Comment (undead404) [inline client/src/components/WaterQualitySensor/WaterQualitySensor.js]: Please rename it to start with `render` word

**Review by undead404 — CHANGES_REQUESTED** [review]

File: client/src/components/WaterQualitySensor/WaterQualitySensor.js [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+          <div className={style.filters}>
+            {this.filterButtons.map(button => {
+              return (
+                <Button
+                  className={classNames(style.filterBtn, {
+                    [style.filterBtnActive]: filter === button.value,
+                  })}
+                  variant="outline-info"
+                  key={button.id}
+                  onClick={dispatchChangeFilter(button.value)}
```
Comment (undead404) [inline client/src/components/WaterQualitySensor/WaterQualitySensor.js]: Hmmm, does this work?

Comment (ArtemOliinyk) [reply client/src/components/WaterQualitySensor/WaterQualitySensor.js]: It does, why not?

**Review by undead404 — APPROVED** [review]

File: server/go.mod [hunk 9]
```diff
@@ -3,6 +3,8 @@ module school-project-2019/server
 go 1.12
 
 require (
+	github.com/jasonlvhit/gocron v0.0.0-20190920201010-985d45da66c5
```
Comment (undead404) [inline server/go.mod:6]: Do you use gocron?

**Pull request merged**